ui/sagas/dependencies.js

```diff
@@ -0,0 +1,9 @@
+import { call, put } from 'redux-saga/effects';
+
+import { actions } from '../ducks/dependencies';
+import { get } from '../api';
+
+export function* fetchScripts() {
+  const { dependencies } = yield call(get, '/dependencies');
```

Comment:
I think you just want `const dependencies` here as otherwise you drop the `devDependencies`